test-integration/samples/webflux-gradle-session-h2mem-es/.yo-rc.json

```
@@ -21,7 +21,7 @@
     "prodDatabaseType": "mariadb",
     "reactive": true,
     "rememberMeKey": "a5e93fdeb16e2ee2dc4a629b5dbdabb30f968e418dfc0483c53afdc695cfac96d06cf5c581cbefb93e3aaa241880857fcafe",
-    "searchEngine": "elasticsearch",
+    "searchEngine": false,
```

Reenable elastic search
```suggestion
    "searchEngine": "elasticsearch",
```